lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -18,6 +18,8 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
 
     defstruct station: nil
 
+    # TODO: just curious, do the separate `alert_ids` definitions lead to double-counting
```

Comment:
I was very curious as well so I looked into this. An `alert_id` is counted twice if it has a `DetailPage` and is also on a `ListPage`, but the duplicates are filtered on line 553.